src/effect-store.ts

```diff
@@ -1,33 +1,27 @@
-/**
+/*
```

Comment:
In some places I had inadvertently (or lazily) use `/**` which makes blocks more readily, but gets consumed as docstrings for the next variable, which was not the intention.